third_party/2and3/Crypto/Random/random.pyi

```diff
@@ -1,17 +1,19 @@
-from typing import Any, Optional
+from typing import Any, List, Optional, Sequence, TypeVar
+
+_T = TypeVar('_T')
 
 class StrongRandom:
     def __init__(self, rng: Optional[Any] = ..., randfunc: Optional[Any] = ...) -> None: ...
-    def getrandbits(self, k): ...
-    def randrange(self, *args): ...
-    def randint(self, a, b): ...
-    def choice(self, seq): ...
-    def shuffle(self, x): ...
-    def sample(self, population, k): ...
+    def getrandbits(self, k: int) -> int: ...
+    def randrange(self, *args: int) -> int: ...
+    def randint(self, a: int, b: int) -> int: ...
+    def choice(self, seq: Sequence[_T]) -> _T: ...
+    def shuffle(self, x: Sequence[_T]): ...
+    def sample(self, population: Sequence[_T], k: int) -> List[_T]: ...
 
-getrandbits: Any
-randrange: Any
-randint: Any
-choice: Any
-shuffle: Any
-sample: Any
+def getrandbits(self, k: int) -> int: ...
+def randrange(self, *args: int) -> int: ...
+def randint(self, a: int, b: int) -> int: ...
+def choice(self, seq: Sequence[_T]) -> _T: ...
+def shuffle(self, x: Sequence[_T]): ...
+def sample(self, population: Sequence[_T], k: int) -> List[_T]: ...
```

Comment:
Since these are bound methods, the `self` argument is transparent to the caller and should not be included here. Also see comments above about `shuffle`.

Reply:
fixed